src/capi_frontend/capi_utils.cpp

```diff
@@ -0,0 +1,71 @@
+//*****************************************************************************
+// Copyright 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "capi_utils.hpp"
+
+#include <map>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+
+#include "../logging.hpp"
+#include "../pocapi.hpp"
+#include "../precision.hpp"
+#include "../profiler.hpp"
+#include "../shape.hpp"
+#include "../status.hpp"
+
+namespace ovms {
+
+size_t DataTypeSize(const OVMS_DataType& datatype) {
```

Comment:
This is already implemented in inferenceparameter.cpp. It should be moved out of there